src/Datadog.Trace.ClrProfiler.Managed/Integrations/ElasticsearchNet6Integration.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.Integrations
+{
+    /// <summary>
+    /// Traces an Elasticsearch pipeline
+    /// </summary>
+    public static class ElasticsearchNet6Integration
+    {
+        // NOTE: keep this name without the 6 to avoid breaking changes
+        private const string IntegrationName = "ElasticsearchNet";
```

Comment:
So, renaming the class in our integrations, and keeping the name the same is safe, in terms of breaking changes?
i.e., we should never run into a consumer directly referencing our `ElasticsearchNet6Integration` class, right?

Reply:
Right, but integration name (`ElasticsearchNet`) is used for configuration.